integreat_cms/cms/models/events/event.py

```diff
@@ -44,16 +45,18 @@ def filter_upcoming(self, from_date: datetime | None = None) -> EventQuerySet:
         :param from_date: The date which should be used for filtering, defaults to the current date
         :return: The Queryset of events after the given date
         """
-        from_date = from_date or timezone.now().date()
+        from_date = (from_date or timezone.now()).min.replace(
+            tzinfo=ZoneInfo(key="UTC")
+        )
```

Comment:
```suggestion
        from_date = from_date or timezone.now()
```
Hmm, this was not exactly what I meant, I meant to call the `.replace(tzinfo=ZoneInfo(key="UTC"))` [here](https://github.com/digitalfabrik/integreat-cms/blob/6e60e28067c1f525019e12e6e8671060e9784c46/integreat_cms/cms/forms/events/event_filter_form.py#L102) where the actual `OverflowError` is happening. It's only needed for the fallback value of `date.min` in combination with `time.min`. The other times are fine, Django is able to automatically convert between timezones, as long as the given datetime is timezone-aware (which `timezone.now()` is).